ticdc/ticdc-data-replication-capabilities.md

```diff
@@ -45,4 +45,6 @@ TiCDC supports the following types of upstream data changes:
 ## Limitations
 
 - TiCDC does not support certain scenarios. For details, see [unsupported scenarios](/ticdc/ticdc-overview.md#unsupported-scenarios).
-- TiCDC only verifies the integrity of upstream data changes. It does not validate whether the changes conform to upstream or downstream constraints. If the data violates downstream constraints, TiCDC will return an error when writing to the downstream. For example, TiCDC does **not** perform any foreign key validation.
+- TiCDC only verifies the integrity of upstream data changes. It does not validate whether the changes conform to upstream or downstream constraints. If the data violates downstream constraints, TiCDC will return an error when writing to the downstream. 
+
+   For example: When a changefeed is configured to filter out all DDL events, if the upstream executes a `DROP COLUMN` DDL and continues to write `INSERT` statements involving that column, TiCDC will fail to synchronize these DML changes downstream due to divergent table schemas, resulting in data write failures.
```

Comment:
```suggestion

    For example: When a changefeed is configured to filter out all DDL events, if the upstream executes a `DROP COLUMN` operation but continues to write `INSERT` statements involving that column, TiCDC will fail to replicate these DML changes to the downstream because of table schema mismatches.
```